### PR TITLE
Csp add resrouces

### PR DESCRIPTION
**Pull request by ricardoMogg**
**What changed? Why?**
Added csp entries for fonts-src default-src a connect-src

This should reduce many of these issues:
https://app.datadoghq.com/rum/performance-monitoring?query=%40session.type%3Auser%20%40application.id%3A16db4f8f-831a-4df5-b78b-546405085227&filters=%5B%22env%22%2C%22service%22%2C%22version%22%2C%22%40session.type%22%2C%22%40geo.country%22%2C%22%40device.type%22%2C%22%40usr.email%22%5D&fromUser=false&fullscreen_end_ts=1724792312000&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_start_ts=1724788712000&tab=errors&view=spans&from_ts=1724788712164&to_ts=1724792312164&live=true

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: apps/web/next.config.js [hunk 1]
```diff
@@ -116,6 +121,7 @@ const contentSecurityPolicy = {
     "'self'",
     'blob:',
     'data:',
+    'https://euc.li',
```
Comment (ricardoMogg) [inline apps/web/next.config.js:124]: Screenshot 2024-08-27 at 2 48 33 PM

File: apps/web/next.config.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ const walletconnectDomains =
   'https://*.walletconnect.org wss://*.walletconnect.org wss://*.walletconnect.com https://*.walletconnect.com https://explorer-api.walletconnect.com';
 
 const contentSecurityPolicy = {
+  'font-src': ['https://fonts.gstatic.com'],
```
Comment (ricardoMogg) [inline apps/web/next.config.js:71]: Screenshot 2024-08-27 at 2 30 57 PM

File: apps/web/next.config.js [hunk 3]
```diff
@@ -77,7 +78,7 @@ const contentSecurityPolicy = {
     ccaLiteDomains,
     walletconnectDomains,
   ],
-  'worker-src': ["'self'"],
+  'worker-src': ["'self'", 'blob:'],
```
Comment (ricardoMogg) [inline apps/web/next.config.js:81]: Screenshot 2024-08-27 at 2 51 05 PM

**Review by zencephalon — APPROVED** [review]

**Pull request merged**